src/java.base/share/classes/java/lang/foreign/package-info.java

```diff
@@ -31,14 +31,12 @@
  *
  * <p>
  * The main abstraction introduced to support foreign memory access is {@link java.lang.foreign.MemorySegment}, which
- * models a contiguous memory region, which can reside either inside or outside the Java heap.
- * A memory segment represents the main access coordinate of a memory access var handle, which can be obtained
- * using the combinator methods defined in the {@link java.lang.invoke.MethodHandles} class; a set of
- * common dereference and copy operations is provided also by the {@link java.lang.foreign.MemorySegment} class, which can
- * be useful for simple, non-structured access. Finally, the {@link java.lang.foreign.MemoryLayout} class
- * hierarchy enables description of <em>memory layouts</em> and basic operations such as computing the size in bytes of a given
- * layout, obtain its alignment requirements, and so on. Memory layouts also provide an alternate, more abstract way, to produce
- * memory access var handles, e.g. using <a href="MemoryLayout.html#layout-paths"><em>layout paths</em></a>.
+ * models a contiguous memory region, which can reside either inside or outside the Java heap. Moreover,
+ * the {@link java.lang.foreign.MemoryLayout} class hierarchy enables description of <em>memory layouts</em>
```

Comment:
enables **the** description of <em>memory layouts</em> and basic operations such as computing the size in bytes of a given layout **or** obtaining its alignment requirements?